workloads/kube-burner-ocp-wrapper/metrics-profiles/hosted-cluster-metrics.yml

```diff
@@ -31,6 +31,22 @@
 - query: (node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes) and on (instance) label_replace(kube_node_role{role="infra"}, "instance", "$1", "node", "(.+)")
   metricName: nodeMemoryUtilization-Infra
 
+- query: node_memory_MemAvailable_bytes{} and on (instance) label_replace(kube_node_role{role="worker"}, "instance", "$1", "node", "(.+)")
+  metricName: nodeMemoryAvailable-Workers
+
+- query: node_memory_MemTotal_bytes{} and on (instance) label_replace(kube_node_role{role="worker"}, "instance", "$1", "node", "(.+)")
+  metricName: nodeMemoryTotal-Workers
+
+- query: (max(sum(irate(node_cpu_seconds_total{}[2m])) by (mode,instance) and on (instance) label_replace(bottomk(1, min_over_time(sum(irate(node_cpu_seconds_total{mode=~"idle"}[2m])) by (mode,instance)[{{ .elapsed }}:])), "instance", "$1", "instance", "(.+)")) by (mode)) > 0
+  metricName: nodeCPU-AggregatedWorkers
```

Comment:
I think this metric should be renamed to something like `nodeCPU-MostUtilizedWorker`